newsroom/auth/views.py

```diff
@@ -255,3 +255,11 @@ def set_locale():
     if locale and locale in app.config["LANGUAGES"]:
         flask.session["locale"] = locale
     return flask.redirect(flask.url_for("auth.login"))
+
+
+@blueprint.route("/login/<client>", methods=["GET"])
```

Comment:
Would this be better off in `newsroom.auth.saml` file instead, so we're not adding an endpoint that has no use if SSO is not enabled?